test/notifications/open_test.dart

```diff
@@ -116,7 +113,12 @@ void main() {
         return;
       }
       await tester.pump();
-      takeStartingRoutes(withAccount: withAccount);
+      final lastVisitedAccount = testBinding.globalStore.lastVisitedAccount;
+      if (lastVisitedAccount != null) {
+        takeHomePageRouteForAccount(lastVisitedAccount.id);
+      } else {
+        takeChooseAccountPageRoute();
```

Comment:
nit: this conditional gets introduced in one commit, and then the order is swapped around in a later commit; better to simplify the diff in the later commit by having it not change the order.

(I think either order is fine; either have the earlier commit go straight to this order, or have the later commit leave it in the other order.)